README.md

```diff
@@ -86,6 +86,35 @@ Uses the same options as [`createContentType`](#createcontenttypeid--string-opts
 
 Deletes the content type with the provided id and returns `undefined`. Note that the content type must not have any entries.
 
+#### `transformEntries(config)`
+
+For the given content type, transforms all entries according to the user-provided `transformEntryForLocale` function. This function will be called for every entry and every locale with the `from` fields and the current locale. It is expected to return an object with the desired target fields and the value OR undefined, in which case this entry locale will be skipped. If `transformEntryForLocale` does return a value, it will be written to the current locale.
```

Comment:
I would also change the line `This function will be called for every entry and every locale with the `from` fields and the current locale.` to `For each entry, the cli will call this function once per locale in the space with ...`

Reply:
```diff
- cli
+ CLI
```